seeder/index.js

```diff
@@ -1,13 +1,22 @@
 // Seeding file. Executing this file will seed the database.
 
-const seed = require('./seeder');
 const db = require('../database-mysql');
+// new code
+const fs = require('fs');
 
 const names = ['Bob', 'Charles', 'Jack', 'Jill', 'DrNy', 'Chris', 'Sarah', 'Kim', 'Mary', 'xxAio', 'giant', 'F.O', 'M.A', 'W.W', 'L.L', 'Arkoo', 'Mark', 'Lauren', 'Young', 'Rebecca'];
 const dates = ['Aug 08, 2020', 'Jan 01, 2020', 'Feb 12, 2020', 'Mar 22, 2020', 'Sep 11, 2020', 'Jul 02, 2020', 'Oct 29, 2020', 'Mar 10, 2020', 'Apr 05, 2020', 'Jun 03, 2020', 'Jun 05, 2020', 'May 05, 2020', 'May 25, 2020'];
+const seeder = require('../seeder/seeder')
+const { insertProducts, insertReviews } = require('../database-mysql/index');
+
+seeder.genProducts();
+seeder.genReviews(names, dates);
 
-seed(names, dates);
 
 db.connection.end();
 
 console.log('Seed Finished');
+module.exports = {
+  names,
+  dates
+}
```

Comment:
Maybe stick names & dates in a separate file if you need them in other places? Since if you end up importing them into other documents from this file, it's going to run the seeding generators at line 12-13. Unless you can explain the use case otherwise.

Reply:
I actually don't need them exported anymore, this was a remnant of another strategy I decided not to go with. Exports has been removed.